rust/libdnf-sys/cxx/libdnf.cxx

```diff
@@ -19,6 +19,7 @@
  */
 
 #include "libdnf.hxx"
+#include "libdnf-sys/lib.rs.h"
```

Comment:
It took me a while to figure this bit out. First because I thought that somehow happened automagically since we haven't needed it so far (but in fact, that's because none of the types we were using in this file yet came from cxx.rs), and second because we're missing a bunch of `cargo:rerun-if-changed` in `libdnf-sys/build.rs` which caused me to hit the classic not-testing-what-you-think-you-are issue. Will fix that in a separate PR!